admin/.env.example

```
@@ -1,19 +1,19 @@
 # copy to .env.local for development
+
+# === 必須 ===
+DATABASE_URL=postgresql://postgres:postgres@127.0.0.1:54332/postgres
+
 SUPABASE_URL=http://127.0.0.1:54321
-SUPABASE_ANON_KEY="your anon key from supabase status"
-SUPABASE_SERVICE_ROLE_KEY="your service key from supabase status"
+SUPABASE_ANON_KEY=your anon key from supabase status
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)